### DISCUSSION TITLE
Preset group:pixijsMonorepo does not work

### DISCUSSION THREAD

**Discussion by latin-1** [renovatebot/renovate#21394] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

PixiJS now has a different repo url than the one in the preset.

Repo: https://github.com/pixijs/pixijs

Preset `monorepo:pixijs`:

```json
{
  "matchSourceUrls": [
    "https://github.com/pixijs/pixi.js"
  ]
}
```

Source code: https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/monorepo.ts#L178

**Comment by latin-1** [comment]
@rarkins I believe built-in presets should follow the repo renaming.

**Reply by rarkins** [reply]
PRs welcome!

**Reply by HonkingGoose** [reply]
We got the PR:

- https://github.com/renovatebot/renovate/pull/21396

**Comment by viceice** [comment]
do we still need the workaround?
